Production/govuk_ios/ViewControllers/RecentActivityListViewController.swift

```diff
@@ -261,6 +276,10 @@ class RecentActivityListViewController: BaseViewController,
         ]
         editingToolbar.setItems(items, animated: animated)
     }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
Do we need to call super here like we've done everywhere else we've overridden this function?